pages/guideforms.py

```diff
@@ -57,7 +57,7 @@ def to_python(self, value):
         # Return an empty list if no input was given.
         if not value:
             return []
-        return value.split('\n')
+        return value.split(r'\s+')
 
     def validate(self, value):
```

Comment:
We discussed removing the sever-side validation today.   However, I don't think that we can do that quite yet.  One reason why this validation is here is to prevent against an attacker inputting a URL with the `javascript:` scheme.  That won't make it past client-side validation, but an attacker could use `wget` to make a POST request to our server rather than using a browser.  So, before we can remove this, we need to add defensive validation at every place where we currently output a URL.